core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -675,9 +680,12 @@ public FileInfo getFileInfo(long fileId)
   public FileInfo getFileInfo(AlluxioURI path, GetStatusOptions options)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException {
     Metrics.GET_FILE_INFO_OPS.inc();
-
+    Inode srcInode = null;
+    RpcUtils.RpcContext rpcContext = RpcUtils.createRpcContext().setAllowed(true);
```

Comment:
We discussed offline and decided on a way to use try-with-resources to make an audit context very similar to the journal context.